gems/pending/util/postgres_admin.rb

```diff
@@ -12,6 +12,10 @@ def self.data_directory
     Pathname.new(ENV.fetch("APPLIANCE_PG_DATA"))
   end
 
+  def self.template_directory
+    Pathname.new(ENV.fetch("APPLIANCE_TEMPLATE_DIRECTORY"))
+  end
```

Comment:
This feels really weird to have a PostgresAdmin guy reference something in application system files.  Just feels like the wrong place for it.